roles/openshift_on_openstack_scaleup/vars/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+# A list of regular expressions to search and replace in openshift-ansible all.yml.
+all_yml:
+  - { find: "^openshift_openstack_num_masters.*", replace: "openshift_openstack_num_masters: 3" }
+  - { find: "^openshift_openstack_num_nodes.*", replace: "openshift_openstack_num_nodes: 5" }
+  - { find: "^openshift_openstack_num_infra.*", replace: "openshift_openstack_num_infra: 3" }
```

Comment:
A scale up operation only adds nodes. Why do you replace the infra here? (These values are __already__ 3 from the openshift_on_openstack role.)

Reply:
Again, feel free to remove.